app/assets/stylesheets/application.css

```diff
@@ -28,3 +28,25 @@
   text-align: center;
   padding: 20rem 10rem;
 }
+
+nav {
+  position: sticky;
+  top: 0;
+  z-index: 50;
+}
+
+nav {
+  background-color: white;
+  border: 1px solid whitesmoke;
+  padding: 1rem;
+}
+
+.rounded-pill {
```

Comment:
I need to hurry about and integrate SCSS, but eventually we should separate component styling (i.e. have them in their own SCSS file)